Week2/homework/index.js

```diff
@@ -0,0 +1,114 @@
+console.log(1);
+let HiIndo='Hallo, dunia!'
+let HiIta='Ciao, mondo!' 
+let HiSpa='Hola, mundo!'
+let HiRus='Priviet, mir!'
+let HiKaz='Salamtsizba, duniye!'
+let HiDutch='Hallo, wereld!'
+let HiFr='Bonjour, le monde!'
+let HiTr='Merhaba, dunya!'
+let HiGr='Hallo, welt!'
+let HiRomanian='Salut, lume!'
+console.log(HiIndo);
+console.log(HiIta);
+console.log(HiSpa);
+console.log(HiRus);
+console.log(HiKaz);
+console.log(HiDutch);
+console.log(HiFr);
+console.log(HiTr);
+console.log(HiGr);
+console.log(HiRomanian);
+
+console.log(2);
+console.log('I'm awesome');
+// Uncaught SyntaxError: missing ) after argument list
+//The symbol used for the string is the same with the symbol used in the syntax of the string.
+// When I changed it into sth else, i gives what i wanted to write.
+console.log(" I'm awesome!");
+
+
+console.log(3)
+let x;
+x=3;
+
+console.log("the value of my variable x will be: 3");
+console.log(x);
+
+
+console.log(4);
+var y="I LOve Spring";
+console.log("The value of my variable y will be: I Love Spring!);
+
+var y="The Rain Is Beautiful!";
```

Comment:
I don't think this will work, because you already defined `y` at line 40. Please recheck this line

Reply:
Hi,
i knew about  this actually. 
i don't know why i did it so.
i have corrected.
Thank you .